integration-tests/smoke/ccip/ccip_reader_test.go

```diff
@@ -364,7 +363,6 @@ func TestCCIPReader_CommitReportsGTETimestamp_RespectsFinality(t *testing.T) {
 	require.Never(t, func() bool {
 		reports, err = s.reader.CommitReportsGTETimestamp(
 			ctx,
-			chainD,
 			// Skips first report
```

Comment:
There a reason we removed the `destChain` argument from this?

Reply:
I think this was from a change @dimkouv made. Not sure why it is a diff in this PR, maybe needs a rebase.

Reply:
Yes, I updated the go version for chainlink-ccip, but the integration test needs to be updated. I will resolve this in the feature branch.